pkg/boot/image/image.go

```diff
@@ -23,7 +23,7 @@ var (
 	kernelImageSize = uint64(math.Pow(2, 24)) // 16MB, a guess value similar to that used in kexec-tools.
 
 	errBadMagic     = errors.New("bad header magic")
-	errBadEndianess = errors.New("invalid Image endianess, expected little")
+	errBadEndianness = errors.New("invalid Image endianness, expected little")
```

Comment:
This file needs to be formatted with gofmt tool.

Reply:
this command will do the trick:
```
find . -type f -name "*.go" ! -path "./vendor/*" -exec gofmt -w {} \;
```

Reply:
Thank you. I did it and committed the result.